docs/index.md

```diff
@@ -294,101 +337,344 @@ Follow these steps:
-	vault lease revoke astra/org/token/UuS2JYK999999999999Ip3v1
+    Key                Value
+    ---                -----
+    lease_id           astra/org/token/ove4ll9sshXFL3qK6I1qPezb
+    lease_duration     1h
+    lease_renewable    true
 	```
 
-13. Delete tokens based on an Astra DB `org_id`, `role_name`, and `logical_name`. Example:
+    *NOTE*: The token can be renewed to a value no greater than `max_ttl`.
+
+
+## Upgrade plugin from binary distribution
+
+Plugin versioning was introduced in Vault 1.12, allowing for a smooth upgrade of the plugin that has been mounted at a path on a running Vault server. These steps assume you have already registered the plugins as outlined under "Setup plugin from binary distribution".
+
+1. To upgrade the plugin, register a newer version of the plugin. You **must** use the same plugin type (secret) and name (vault-plugin-secrets-datastax-astra) as the plugin being upgraded:
 
 	```bash
-	vault delete astra/org/token org_id="ccd999999_facd_4ad3_bbb99903d999999999999999d" \
-	role_name="organization_administrator" logical_name="org_logical_name"
+	vault plugin register -sha256=${SHA256} \
+	  -command=vault-plugin-secrets-datastax-astra_1.0.1 \
+	  -version=v1.0.1 \
+	  secret vault-plugin-secrets-datastax-astra
 	```
 
-	**Output:**
+2. Tune the existing mount to configure it to use the newly registered version:
+
+	```bash
+	vault secrets tune -plugin-version=v1.0.1 vault-plugin-secrets-datastax-astra
+	```
+
+3. If you want, you can check the updated configuration. Notice the "Version" is now different from the "Running Version":
+
+	```bash
+	vault secrets list -detailed
 	```
-	Success! Data deleted (if it existed) at: astra/org/token
+
+4. As the final step, trigger a plugin reload. This will reload all mounted backends using that plugin:
+
+	```bash
+	vault plugin reload -plugin vault-plugin-secrets-datastax-astra
 	```
 
 ## Summary
 
-HashiCorp Vault has a full understanding of the historical token specifics, for control and auditing purposes, including when the tokens were used and by whom, along with a free-form role name and any custom metadata you may have associated with the tokens. For example, in the example above, HashiCorp Vault's data knows the details of the token delete operation via its identity management and access control data; whereas Astra DB (in this example) is only aware that a token of a particular clientId was generated on a date, and has since been deleted.
+HashiCorp Vault has a full understanding of the historical token specifics, for control and auditing purposes, including when the tokens were used and by whom, along with a free-form role name and any custom metadata you may have associated with the tokens. For example, HashiCorp Vault's data knows the details of the token delete operation via its identity management and access control data; whereas Astra DB is only aware that a token of a particular `clientId` was generated on a date, and has since been deleted.
```

Comment:
```suggestion
HashiCorp Vault has a full understanding of the historical token specifics, for control and auditing purposes, including when the tokens were used and by whom, along with a free-form role name and any custom metadata you may have associated with the tokens. For example, HashiCorp Vault's data knows the details of the token delete operation through its identity management and access control data; whereas Astra DB is only aware that a token of a particular `clientId` was generated on a date, and has since been deleted.
```